### ISSUE TITLE
Username and password required for github.com

### ISSUE THREAD

**Issue by Railwaymaniac** [JosefJantzen/MMM-AlexaControl#10]
Hello
I'm very new to github, and I'm not sure what's going on here - I hope you can help please?
I have very recently managed to configure myself a MagicMirror and wanted to include your control from Alexa.
So I opened a terminal session on my Raspberry Pi, changed to the 'MagicMirror/modules' directory and typed in the 'git clone...' line.
It said 'Cloning into `MM-AlexaControl` and then demanded a username and password for 'https://github.com'.
I tried <blank> and <blank>
I tried <JoChef2> and blank>
I created an account on github for myself and tried my credentials..
None of these work, and the software claims to need a username and password for github.
Do you know anything about this, please??
Ian

**Comment by Railwaymaniac** [comment]
Damn. That ran into HTML, I think. Let me try again.
I tried 'blank' and 'blank'.
I tried 'JoChef2' and 'blank'.
I tried my own credentials. 
And none of these combinations worked.
Can you help, please?

**Comment by JosefJantzen** [comment]
Hi,
you have to clone ``MMM-AlexaControl`` and not ``MM-AlexaControl``. Then isn't an account required.

**Comment by Railwaymaniac** [comment]
Wonderful - many thanks
👍
